Source/Core/Core/IOS/Network/KD/NetKDRequest.h

```diff
@@ -52,11 +51,24 @@ class NetKDRequestDevice : public Device
     return std::nullopt;
   }
 
+  enum ErrorType
+  {
+    ACCOUNT,
+    KD_DOWNLOAD,
+    CLIENT,
+    SERVER,
+  };
+
+  void LogError(ErrorType error_type, s32 error_code);
+
   NWC24::NWC24Config config;
   NWC24::NWC24Dl m_dl_list;
   Common::WorkQueueThread<AsyncTask> m_work_queue;
   std::mutex m_async_reply_lock;
+  std::mutex m_scheduler_buffer_lock;
   std::queue<AsyncReply> m_async_replies;
+  u32 m_error_count;
```

Comment:
```suggestion
  u32 m_error_count = 0;
```

then the assignment in the constructor can be removed